src/index.js

```diff
@@ -51,15 +53,22 @@ function generateCurrentDate() {
   let day = rawDate.getDate();
   if (day < 10) {
     day = `0${day.toString()}`
-  };
+  }
   let month = rawDate.getMonth() + 1;
   if (month < 10) {
     month = `0${month.toString()}`
-  };
+  }
   const year = rawDate.getFullYear();
   return `${year}/${month}/${day}`
 }
 
+function analyzeManagerClick(event) {
```

Comment:
Since there are a few buttons the manager can click, you could add in what click this function is analyzing to the name

Reply:
@ErinUntermeyer I'm planning to add onto this function so there will be additional if conditionals here! That's why I left it generic, since this will eventually analyze any click needed for the manager dash.